db/schema.rb

```diff
@@ -109,6 +109,7 @@
     t.string "status", null: false
     t.string "review_requester"
     t.index ["content_id"], name: "index_step_by_step_pages_on_content_id", unique: true
+    t.index ["review_requester"], name: "fk_rails_91a467475a"
```

Comment:
Where does this value come from? It looks automatically generated, unlike anything else in this file.

Reply:
It was autogenerated by the migration to add a foreign key.